### ISSUE TITLE
CAIA Follow Up

### ISSUE THREAD

**Issue by ChristineRoseSteiffer** [department-of-veterans-affairs/tmf-auth-exp-design-patterns#90]
We need to follow up with CAIA to get guidance on three elements within our designs:
1. The additional messaging around keeping their data secure in the Yellow task
2. The gray card format for locked/editable data in the Yellow and Purple tasks
3. The messaging in the success alert message in all three tasks

ACs

- [x] Designs are provided to CAIA for specific review on those three points
- [x] Feedback from CAIA is incorporated into the designs

**Comment by ChristineRoseSteiffer** [comment]
I pulled out our specific questions for CAIA and put them in their own [page](https://www.figma.com/design/2j01RTqCSJRy4lX3eUOiod/AE-Design-Patterns---Prefill?node-id=689-21949&t=G1BuHSMzgfiksKro-0) within our figma file, then commented on the [ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/90140) with a link to those specific questions. I will post any updates I receive to this ticket. Thanks!

**Comment by ChristineRoseSteiffer** [comment]
I've been working with Laura Wilwerth from CAIA on our designs. She left a few comments in our Figma [files](https://www.figma.com/design/2j01RTqCSJRy4lX3eUOiod/AE-Design-Patterns---Prefill?node-id=11-52&t=cI7qfRkwijCcw4HT-0) for us to work on, which have been incorporated into the designs. These updates are:

- Change the wording of auto save alert to what is in production - ‘Your form has been saved. We saved it on August 23, 2024, at 2:39 p.m. EDT.’ across all designs
- Remove wording at the top of form pages for Application/Request ID (again that matches production)
- Update secure data message in Yellow prototype to:
  - Note: To protect your personal information, we don't allow online changes to your name, date of birth, or Social Security number. If you need to change this information for your health benefits, call your VA health facility.  [link] Find your VA health facility
- Locked data view - gray card - call it ‘Current Home Phone Number’ and add in ‘Edit home phone number’ header above
- Confirmation message language should say  “You've updated your mailing address" “We've also made these changes in your VA.gov profile”
- ‘Need Help’ section - just call it ‘application’ and remove ‘form’

I am incorporating these updates into our [QA document](https://docs.google.com/document/d/1xOraL1E992HkkoZruDFdd5b3j6zy4M1Ixw00BGzddmc/edit) for the engineers so that they can have one source of truth for all updates needed in the prototype.

**Comment by ChristineRoseSteiffer** [comment]
We may need to break off the AC for 'changes have been incorporated into the prototype' until next sprint, unless the engineers think they can do them all on Monday/Tuesday next week. We can discuss during our Monday call.

**Comment by beckyphung** [comment]
I reviewed the [QA document](https://docs.google.com/document/d/1xOraL1E992HkkoZruDFdd5b3j6zy4M1Ixw00BGzddmc/edit#heading=h.m7m3p8834fft) and [designs in Figma](https://www.figma.com/design/2j01RTqCSJRy4lX3eUOiod/AE-Design-Patterns---Prefill?node-id=689-21949&t=bB6hGA56uyVzjeN3-0).

I noticed that the **Your mailing address has been updated** content still needs to be updated. Also, I noticed some open questions in the comment threads with Laura. Are we just going to leave them open until the questions/comments are resolved?

**Comment by ChristineRoseSteiffer** [comment]
Thanks for that catch! I noticed it on our Monday review as well. I've updated the designs and QA document with the correct confirmation message language. 

There were a few items that I think I implemented the way Laura was recommending them, but I did want to get confirmation from her. And there is one outstanding question that is a research ticket I created for next sprint (#[95](https://github.com/orgs/department-of-veterans-affairs/projects/1314/views/4?sliceBy%5Bvalue%5D=Sprint+6&pane=issue&itemId=76217759)) which is to look into how users can edit their personal info that is not editable on the website (name, DOB, SSN, etc). We have some instructions around that and we just want to confirm they are correct or if we should present them in a different way. So that one will stay open until we can understand more about that process.
